portfolio/src/main/webapp/calendar.js

```diff
@@ -278,59 +290,105 @@ function getClosestNextHour(nextHour, workHourStartString, workHourEndString) {
     closestHour += ':00';
   }
 
-  return closestHour; 
+  return closestHour;
 }
- 
+
+/**
+ * Sets the default working hour start time based on user picked date
+ * and the current time.
+ * If user picked today to schedule for,
+ * The default working hour start time is the next closest hour,
```

Comment:
It seems a little weird to me to display different working hours on different days. Can we not use the same working hours for "today" but then still just make sure we never select a time that's in the past? It just feels like it's going to be a little confusing for the user if they see their working hours change when they choose different days.

Reply:
Done.